docs/index.md

```diff
@@ -5,37 +5,35 @@
 :hidden:
 
 Readme <self>
-User Guide <userguide/userguide>
-Examples <examples/examples>
 Contribute <misc/contributing_link>
+Known Issues <known_issues>
```

Comment:
Does anyone (in particular pipeline guru @Scienfitz ) understand why the Reminder check fails and what we can and should do about it?

Reply:
Might this be an issue since this is coming from a fork? In that case I could quickly push the exact same branch here and then create a PR from that one.

Reply:
might be fork or some temporary downside from some web functionality, theres no indication that its to do anything with the code changed and the check is optional anyway